src/partials/hero.html

```diff
@@ -5,16 +5,13 @@ <h2 class="hero-text">Treat yourself or a loved one to<br/> our finest ingredien
         <div class="hero-buttons-all">
             <div class="hero-buttons">
                 <button type="button" class="hero-button-buy" data-modal-open>Buy now</button>
-                <a href="#how-its-made">
-                    <button type="button" class="hero-button-how">How it's made</button>
+                <a href="#how-its-made" class="hero-button-how">How it's made>
                 </a>
             </div>
-            <a href="#footer">
-                <button type="button" class="hero-button-scroll">scroll down
+            <a href="#footer" class="hero-button-scroll">scroll down
                     <svg class="icon-arrow">
                         <use href="./images/icons2.svg#icon-arrow"></use>
                     </svg>
-                </button>
             </a>
```

Comment:
to /a daj nad /button

Reply:
czekaj, kasowałam to chyba, zaraz zobacze czy mi dobrze załadowało